bin/skel.py

```diff
@@ -153,20 +153,29 @@ def new_problem():
     author = config.args.author if config.args.author else _ask_variable_string("author")
 
     validator_flags = ""
-    if config.args.validation:
-        parse_validation(config.args.validation)
-        validation = config.args.validation
+    custom_output = False
+    if config.args.type:
+        problem_type = config.args.type
     else:
-        validation = _ask_variable_choice("validation", ["default", "float", "custom"])
-        if validation == "float":
-            validation = "default"
-            validator_flags = "validator_flags:\n  float_tolerance 1e-6\n"
-            log("Using default float tolerance of 1e-6")
-        if validation == "custom":
-            if _ask_variable_bool("interactive", False):
-                validation += " interactive"
-            if _ask_variable_bool("multi-pass", False):
-                validation += " multi-pass"
+        problem_type = _ask_variable_choice("type", ["default", "float", "custom"])
+    # TODO Maybe remove command-line option "default" and replace it with "pass-fail"?
+    if problem_type == "default":
+        problem_type = "pass-fail"
```

Comment:
I don't expect that people have `bt new_problem --validation default` in some automation script anywhere, so I think it's safe to remove? Although I don't mind leaving in the `default` problem type and simply replace that with the new `pass-fail`, similar to how we have the `float` problem type to add the `validator_flags`, and the `custom` problem type to make sure the `output_validators/` folder from the skeleton is included :slightly_smiling_face:

Reply:
Removed it! :sparkles: